package.json

```diff
@@ -24,7 +24,7 @@
     "coverage:system-tests":
       "lerna run coverage --scope system-tests --concurrency 1 --stream",
     "vscode:package":
-      "lerna exec --scope @salesforce/salesforcedx-* -- npm prune --production && lerna run vscode:package --concurrency 1",
+      "lerna exec -- find . -name @types -type d -exec shx rm -rf {} + && lerna exec --scope @salesforce/salesforcedx-* -- npm prune --production && lerna run vscode:package --concurrency 1",
```

Comment:
Don't use find. It's behavior might not be standard across different OS.

Won't `lerna --exec shx rm -rf node_modules/@types` do the trick?

Or am I missing something?

Reply:
That doesn't remove one in system-tests. Wanted to be thorough in case others that actually affect the packaging slip through in the future.

Reply:
Ok does `find` work on both Linux and Mac? I don't think find works on windows. So you will fail the appveyor build immediately.

Reply:
Yes works on both but I'll change to your suggestion. I think it's worth having this work on windows even if none of us package there. If people run into issues we can resurface this then.